docs/core/sdk/file-based-apps.md

```diff
@@ -103,7 +104,7 @@ Compile your file-based app by using the `dotnet build` command:
 dotnet build file.cs
 ```
 
-The SDK generates a temporary project and builds your application.
+The SDK generates a virtual project and builds your application. The default path for the build output is `./bin/<configuration>/<framework>/`. Use the `--output` option to specify a different path.
```

Comment:
The default output path is in the user profile directory and generally isn't visible to the user by default (it's in the directory cleared by the `dotnet clean file-based-apps` command, e.g. for Windows `C:\Users\<username>\AppData\Local\Temp\dotnet\runfile\<appname>-<appfilesha>\bin\debug\`). Easiest way to see it is by using the `-getProperty` option, e.g.:

```shell
> dotnet build hello.cs -getProperty=OutputPath
C:\Users\username\AppData\Local\Temp\dotnet\runfile\hello-1b946725adaab7fbf948510079efbd804fd48dbeeb8f1fa765bb7197dd8d5e73\bin\debug\
```

Using `--output` will change the output path for that build invocation, or the user can set the `OutputPath` property in their app using the directive:

```csharp
#:property OutputPath=./output
```